frontend/src/components/LoginForm/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import TextField from '../TextField';
+import Button from '../Button';
+import classnames from 'classnames/bind';
+import styles from './LoginForm.module.scss';
+
+const cx = classnames.bind(styles);
+
+class LoginForm extends Component {
+  render() {
+    return( 
+      <div className={cx('login-box')}>
+        <p>짜요짜요</p>
+        <div className={cx('form-contents')}>
+          <TextField
+            name="id"
+            label="이메일"
+          />
+          <TextField
+            name="password"
+            label="비밀번호"
+          />
+        </div>
+        <div className={cx('sign-btn')}>
+          <Button 
+            children = "로그인"
```

Comment:
보통 children 프로퍼티는 태그내로 감싸는게 좋을 것 같아요. 다른 곳과 syntax를 맞출 수 있었어요!
```suggestion
<Button>로그인</Button>
```